azurelinuxagent/pa/deprovision/default.py

```diff
@@ -266,3 +270,21 @@ def del_persist_firewall_rules(actions):
         actions.append(DeprovisionAction(fileutil.rm_files,
                                          [agent_network_service_path, os.path.join(conf.get_lib_dir(),
                                           PersistFirewallRulesHandler.BINARY_FILE_NAME)]))
+
+    @staticmethod
+    def remove_agent_cgroup_config(actions):
+        # Get all service drop in file paths
+        agent_drop_in_path = systemd.get_agent_drop_in_path()
+        slice_path = os.path.join(agent_drop_in_path, _AGENT_DROP_IN_FILE_SLICE)
+        cpu_accounting_path = os.path.join(agent_drop_in_path, _DROP_IN_FILE_CPU_ACCOUNTING)
+        cpu_quota_path = os.path.join(agent_drop_in_path, _DROP_IN_FILE_CPU_QUOTA)
+        mem_accounting_path = os.path.join(agent_drop_in_path, _DROP_IN_FILE_MEMORY_ACCOUNTING)
+
+        # Get slices to remove
+        unit_file_install_path = systemd.get_unit_file_install_path()
+        log_collector_slice_path = os.path.join(unit_file_install_path, LOGCOLLECTOR_SLICE)
+        azure_slice_path = os.path.join(unit_file_install_path, AZURE_SLICE)
```

Comment:
I wouldn't remove this file too because we are leaving azure.slice definition in service file in new agent versions. Otherwise, it may produce issues.

Reply:
can you update PR description too for future reference?